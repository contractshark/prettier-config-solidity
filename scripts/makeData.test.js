require('jest-mock-now')();
const makeData = require('./makeData');

test('makeData to match snapshot', () => {
  expect(makeData('../src')).toMatchSnapshot();
});
