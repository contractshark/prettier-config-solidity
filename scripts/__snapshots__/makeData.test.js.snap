// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`makeData to match snapshot 1`] = `
"/* This file was automatically generated on 1479427200 */

  /* eslint-disable global-require */

  module.exports = {};"
`;
